### PR TITLE
Enable Python migration utility button always and modify the tooltip message.

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

Task: https://jira.autodesk.com/browse/DYN-4371

Enabled the Python migration utility button always and modified the tooltip message. @QilongTang @Amoursol @mjkkirschner We can do this if this would be more clear to the users.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Enable the Python migration utility button always.

### Reviewers
@QilongTang

### CODE REVIEW

File: src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml [hunk 1]
```diff
@@ -56,7 +56,6 @@
                 <Setter Property="Foreground"
                         Value="#bbbbbb" />
             </Style>
-            <pw:SelectedPythonEngineToMigrationConverter x:Key="SelectedPythonEngineToMigrationConverter" />
```
Comment (reddyashish) [inline src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml:59]: SelectedPythonEngineToMigrationConverter class is not being used currently but could be useful in the future. Do we want to remove the class as well now?

File: src/Libraries/PythonNodeModels/Properties/Resources.resx [hunk 2]
```diff
@@ -192,7 +192,7 @@
     <value>Use this dropdown to choose the python version/engine to execute your code.</value>
   </data>
   <data name="PythonScriptEditorMigrationAssistantButtonTooltip" xml:space="preserve">
-    <value>Migration Assistant to update your Python 2 scripts to Python 3. Engine must be set to Python 2 version to work.</value>
+    <value>Run migration utility tool to make the script Python 3 compliant.</value>
```
Comment (aparajit-pratap) [inline src/Libraries/PythonNodeModels/Properties/Resources.resx:195]: Is this because IronPython2 engine is not present by default?

Comment (reddyashish) [reply src/Libraries/PythonNodeModels/Properties/Resources.resx:195]: yes

File: src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml (hunk 1 above)
Comment (QilongTang) [reply src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml:59]: @reddyashish Can you remind us what does the converter do?

Comment (reddyashish) [reply src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml:59]: Just a boolean converter to check if the engine version is Python2 https://github.com/DynamoDS/Dynamo/blob/2f9354254b83e9265d45a192a4e6e69c5d629eae/src/Libraries/PythonNodeModelsWpf/ValueConverter.cs#L8

Comment (QilongTang) [reply src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml:59]: I would remove it

**Review by QilongTang — APPROVED** [review]
LGTM with the legacy converter removed

**Pull request merged**